cloud_functions/climateiq_export_to_aws_cf/main.py

```diff
@@ -45,16 +53,14 @@ def export_to_aws(request: flask.Request) -> tuple[str, int]:
         aws_access_key_id=aws_access_key_id,
         aws_secret_access_key=aws_secret_access_key,
     )
-    for blob in blobs_to_export:
+    for i, blob in enumerate(blobs_to_export):
+        logging.info(f"Exporting {blob.name} ({i + 1}/{total_blobs})...")
         with blob.open("rb") as fd:
             s3_client.upload_fileobj(fd, S3_BUCKET_NAME, f"{curr_time_str}/{blob.name}")
             blob.metadata = {"export_time": curr_time_str}
             blob.patch()
-    return (
-        f"Successfully exported {len(blobs_to_export)} CSV files to AWS bucket: "
-        f"{S3_BUCKET_NAME}.\n",
-        200,
-    )
+        logging.info(f"Successfully exported {blob.name} ({i + 1}/{total_blobs}).")
+    return (f"Successfully exported {total_blobs} CSV files to ClimaSens.\n", 200)
```

Comment:
can you also output the entire S3 bucket name in the response? including the curr_time_str